### PR TITLE
Add spec-pretty subcommand to all connectors

### PR DESCRIPTION
**Pull request by ypc-faros**
## Description

Add spec-pretty subcommand to all connectors. This command generates a help table from the connector's spec to guide a user on how to configure the connector via the command line args when using Faros [airbyte-local-cli](https://github.com/faros-ai/airbyte-local-cli)

Example output for a source:
Screenshot 2023-01-25 at 12 30 21 AM

Example output for a destination (truncated):
Screenshot 2023-01-25 at 12 31 40 AM

## Type of change
- [ ] Bug fix
- [X] New feature
- [ ] Breaking change

## Related issues

> Fix [#1]() 

## Migration notes

> Describe migration notes if any

## Extra info

> Add any additional information

### CODE REVIEW

File: faros-airbyte-cdk/src/destinations/destination-runner.ts [hunk 1]
```diff
@@ -26,7 +27,8 @@ export class AirbyteDestinationRunner<
       .version('v' + PACKAGE_VERSION)
       .addCommand(this.specCommand())
       .addCommand(this.checkCommand())
-      .addCommand(this.writeCommand());
+      .addCommand(this.writeCommand())
+      .addCommand(this.specPrettyCommand());
```
Comment (tovbinm) [inline faros-airbyte-cdk/src/destinations/destination-runner.ts:31]: does the order matter here? perhaps do
```
.addCommand(this.specCommand())
.addCommand(this.specPrettyCommand())
.addCommand(this.checkCommand())
.addCommand(this.writeCommand());
```

File: faros-airbyte-cdk/src/destinations/destination-runner.ts [hunk 2]
```diff
@@ -102,6 +104,30 @@ export class AirbyteDestinationRunner<
       );
   }
 
+  specPrettyCommand(): Command {
+    return new Command()
+      .command('spec-pretty')
+      .description('spec-pretty command')
```
Comment (tovbinm) [inline faros-airbyte-cdk/src/destinations/destination-runner.ts:110]: .description('pretty spec command')

File: faros-airbyte-cdk/src/sources/source-runner.ts [hunk 3]
```diff
@@ -25,7 +26,8 @@ export class AirbyteSourceRunner<Config extends AirbyteConfig> extends Runner {
       .addCommand(this.specCommand())
       .addCommand(this.checkCommand())
       .addCommand(this.discoverCommand())
-      .addCommand(this.readCommand());
+      .addCommand(this.readCommand())
+      .addCommand(this.specPrettyCommand());
```
Comment (tovbinm) [inline faros-airbyte-cdk/src/sources/source-runner.ts:30]: same here

Comment (tovbinm) [inline faros-airbyte-cdk/src/sources/source-runner.ts:30]: same as above

File: faros-airbyte-cdk/src/sources/source-runner.ts [hunk 4]
```diff
@@ -110,4 +112,28 @@ export class AirbyteSourceRunner<Config extends AirbyteConfig> extends Runner {
         }
       );
   }
+
+  specPrettyCommand(): Command {
+    return new Command()
+      .command('spec-pretty')
+      .description('spec-pretty command')
```
Comment (tovbinm) [inline faros-airbyte-cdk/src/sources/source-runner.ts:119]: same as above

**Review by tovbinm — APPROVED** [review]
lgtm!

File: faros-airbyte-cdk/src/destinations/destination-runner.ts (hunk 1 above)
Comment (ypc-faros) [reply faros-airbyte-cdk/src/destinations/destination-runner.ts]: Yeah it does, thanks 👍 !

**Pull request merged**